gc/base/MemorySubSpace.cpp

```diff
@@ -919,17 +919,51 @@ MM_MemorySubSpace::systemGarbageCollect(MM_EnvironmentBase* env, uint32_t gcCode
 	}
 
 	if (_collector && _usesGlobalCollector) {
-		/* TODO: This is bogus for multiple memory spaces - should ask the space, not the heap */
-		_extensions->heap->getResizeStats()->setFreeBytesAtSystemGCStart(getApproximateActiveFreeMemorySize());
+#if defined(OMR_GC_IDLE_HEAP_MANAGER)
+		bool invokeGC = true;
```

Comment:
I would move invokeGC outside of defined(OMR_GC_IDLE_HEAP_MANAGER). In this case there is no need to have if(invokeGC) and correspondent brackets under ifdef